docs/deploy/configurations.md

```diff
@@ -64,51 +64,53 @@ Currently, you can set only 1 namespace to watch in this flag. See [this Kuberne
-|webhook-bind-port                      | int                             | 9443            | The TCP port the Webhook server binds to |
-|webhook-cert-dir                       | string                          | /tmp/k8s-webhook-server/serving-certs | The directory that contains the server key and certificate |
-|webhook-cert-file                      | string                          | tls.crt | The server certificate name |
-|webhook-key-file                       | string                          | tls.key | The server key name |
+| Flag                                                                            | Type                            | Default                                    | Description                                                                                                                                    |
+|---------------------------------------------------------------------------------|---------------------------------|--------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| aws-api-endpoints                                                               | AWS API Endpoints Config        |                                            | AWS API endpoints mapping, format: serviceID1=URL1,serviceID2=URL2                                                                             |
+| aws-api-throttle                                                                | AWS Throttle Config             | [default value](#default-throttle-config ) | throttle settings for AWS APIs, format: serviceID1:operationRegex1=rate:burst,serviceID2:operationRegex2=rate:burst                            |
+| aws-max-retries                                                                 | int                             | 10                                         | Maximum retries for AWS APIs                                                                                                                   |
+| aws-region                                                                      | string                          | [instance metadata](#instance-metadata)    | AWS Region for the kubernetes cluster                                                                                                          |
+| aws-vpc-id                                                                      | string                          | [instance metadata](#instance-metadata)    | AWS VPC ID for the Kubernetes cluster                                                                                                          |
+| aws-vpc-tags                                                                    | stringMap                       |                                            | Tags for the Kubernetes cluster VPC
```

Comment:
Maybe also add a callout here:
>Note that if both flags `--aws-vpc-id` and `--aws-vpc-tags` are specified, the controller will ignore the `--aws-vpc-tags` flag, and use the value in`--aws-vpc-id` to fetch the VPC info.

Reply:
Another question, does it accept multiple key:value pairs for the tags filter?

Reply:
yes, it is a string Map. It will accept multiple values.

Reply:
Added the details here as well.